src/main/java/org/togetherjava/discordbot/BotMain.java

```diff
@@ -14,26 +15,26 @@
 
 public class BotMain {
 
-  private static final Logger logger = LoggerFactory.getLogger(BotMain.class);
+  private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
Why use `MethodHandles`? With a proper IDE refactoring function, this should not be necessary if the class name is ever changed. `MethodHandles` is more low-level compared to Reflection and should not be used in a repository aimed at allowing beginner programmers to commit and view code. Using a direct call to `.class` from the name of the class seems easier and better for performance and code readability. However, I do believe the logger name change is better suited to fit conventions.